common/changes/@rushstack/heft/octogonz-heft-rename-ae-config_2020-07-21-00-26.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "Rename .heft/api-extractor.json to api-extractor-task.json to avoid confusion with API Extractor's config file",
```

Comment:
```suggestion
      "comment": "Rename .heft/api-extractor.json to .heft/api-extractor-task.json to avoid confusion with API Extractor's config file",
```